egs/wsj/s5/steps/nnet3/train_rnn.py

```diff
@@ -0,0 +1,657 @@
+#!/usr/bin/env python
+
```

Comment:
There needs to be a copyright notice at the top of these files (just include your own name).